templates/layouts/_base.html

```diff
@@ -14,7 +14,6 @@
 } %}
 
 {% include 'layouts/configs/_languages.html' %}
-{% include 'layouts/configs/_service-links.html' %}
```

Comment:
`_service-links.html` should be removed then?

Reply:
We left it because it'll be needed somewhere else in eq... We don't know where yet.

Reply:
but if it's included it shows up in header element, when you give it url attribute which is the case.

Reply:
It's been removed now, thanks.